app/controllers/api/v0/admin/configs_controller.rb

```diff
@@ -39,6 +43,23 @@ def update
 
         private
 
+        def settings_params
+          has_emails = params.dig(:settings_general, :email_addresses).present?
+          params[:settings_general][:email_addresses][:default] = ApplicationConfig["DEFAULT_EMAIL"] if has_emails
+
+          params.require(:settings_general)&.permit(
```

Comment:
I feel like I've asked this before but I absolutely can't remember, but do we need the `&` here? `params.require` doesn't return `nil` if the key is missing.

```
[21] pry(main)> ActionController::Parameters.new(foo: "bar").require(:nope)
ActionController::ParameterMissing: param is missing or the value is empty: nope
Did you mean?  foo
```

```suggestion
          params.require(:settings_general).permit(
```

Reply:
I think you asked, I think I answered, but the short answer is copy and paste. Since this controller is currently unused and will be removed very soon by @rhymes I will not apply this suggestion but thanks for noticing! 😃